### ISSUE TITLE
[Intellij.2017.Spark.Template] Run plugin with IC version (community version), hit error: org.jetbrains.idea.maven.project doesn't exist.

### ISSUE THREAD

**Issue by wezhang** [Microsoft/azure-tools-for-java#428]
```
D:\src\github\Microsoft\my\azure-tools-for-java\PluginsAndFeatures\azure-toolkit-for-intellij\src\com\microsoft\azure\hdinsight\projects\samples\MavenProjectGenerator.java
Error:(36, 40) java: 程序包org.jetbrains.idea.maven.project不存在
Error:(143, 9) java: 找不到符号
  符号:   类 MavenProjectsManager
  位置: 类 com.microsoft.azure.hdinsight.projects.samples.MavenProjectGenerator
Error:(143, 40) java: 找不到符号
  符号:   变量 MavenProjectsManager
  位置: 类 com.microsoft.azure.hdinsight.projects.samples.MavenProjectGenerator
```

**Comment by ziyel** [comment]
@wezhang 
Hi Wei, please add below lines to file `PluginsAndFeatures/azure-toolkit-for-intellij/azure-toolkit-for-intellij.iml` to fix this temporarily. I will figure out how to fix this permanently.
```xml
<orderEntry type="module-library" scope="PROVIDED">
   <library>
     <CLASSES>
       <root url="jar://$APPLICATION_HOME_DIR$/plugins/maven/lib/maven.jar!/" />
     </CLASSES>
     <JAVADOC />
     <SOURCES />
   </library>
</orderEntry>
```

**Comment by ziyel** [comment]
Fix in https://github.com/Microsoft/azure-tools-for-java/pull/429.

**Comment by ziyel** [comment]
Fix in #429
